research/minigo/minigo.py

```diff
@@ -79,41 +79,23 @@ def bootstrap(estimator_model_dir, trained_models_dir, params):
   dualnet.export_model(estimator_model_dir, bootstrap_model_path)
 
 
-def selfplay(model_name, trained_models_dir, selfplay_dir, holdout_dir, sgf_dir,
-             params):
+def selfplay(selfplay_dirs, selfplay_model, params):
   """Perform selfplay with a specific model.
 
   Args:
-    model_name: The name of the model used for selfplay.
-    trained_models_dir: The path to the model files.
-    selfplay_dir: Where to write the games. Set as 'base_dir/data/selfplay/'.
-    holdout_dir: Where to write the holdout data. Set as
-      'base_dir/data/holdout/'.
-    sgf_dir: Where to write the sgf (Smart Game Format) files. Set as
-      'base_dir/sgf/'.
-    params: An object of hyperparameters for the model.
+    selfplay_dirs: A dict to specify the directories used in selfplay.
```

Comment:
The original implementation is to specify each one individually. Here as selfplay may be called individually, or in the RL pipeline, we use this dict to avoid duplicate codes.